dev-lang/mono-basic/mono-basic-4.8.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mono-env
+
+MY_PV="e31cb702937a0adcc853250a0989c5f43565f9b8"
+
+DESCRIPTION="Visual Basic Compiler and Runtime"
+HOMEPAGE="https://www.mono-project.com/docs/about-mono/languages/visualbasic/"
+SRC_URI="https://github.com/mono/${PN}/archive/e31cb702937a0adcc853250a0989c5f43565f9b8.tar.gz -> ${P}.tar.gz"
```

Comment:
I suppose you want to reuse the variable here ;-P.

Reply:
This is pro style xP, changed.